x11-packages/emacs-pgtk/src-filelock.c.patch

```diff
@@ -0,0 +1,50 @@
+--- a/src/filelock.c	2022-12-13 12:01:35.000000000 +0000
++++ b/src/filelock.c	2022-12-13 12:04:40.657812208 +0000
+@@ -62,11 +62,11 @@
+ /* A file whose last-modified time is just after the most recent boot.
+    Define this to be NULL to disable checking for this file.  */
+ #ifndef BOOT_TIME_FILE
+-#define BOOT_TIME_FILE "/var/run/random-seed"
++#define BOOT_TIME_FILE "/data/data/com.termux/files/usr/var/run/random-seed"
```

Comment:
Please use `@TERMUX_PREFIX@` instead of `/data/data/com.termux/files/usr`, here and below

Reply:
This should be addressed with the latest push, please see: https://github.com/termux/termux-packages/compare/6d25bf570f4a119d1f1b9dc378ced108d1741519..f633789445a3f8d8b2ac41ab7ae8666e499c31eb